README.md

```diff
@@ -15,7 +15,12 @@ different environments, including:
 
 The Functions Framework for Java requires
```

Comment:
I think it might make sense to say "uses" rather than "requires" here, since the next paragraph shows an alternative.

Reply:
SGTM